presto-main/src/main/java/com/facebook/presto/cost/CoefficientBasedStatsCalculator.java

```diff
@@ -136,11 +135,8 @@ public PlanNodeStatsEstimate visitJoin(JoinNode node, Void context)
             PlanNodeStatsEstimate rightStats = sourceStats.get(1);
 
             PlanNodeStatsEstimate.Builder joinStats = PlanNodeStatsEstimate.builder();
-            if (!leftStats.getOutputRowCount().isValueUnknown() && !rightStats.getOutputRowCount().isValueUnknown()) {
-                double rowCount = Math.max(leftStats.getOutputRowCount().getValue(), rightStats.getOutputRowCount().getValue()) * JOIN_MATCHING_COEFFICIENT;
-                joinStats.setOutputRowCount(new Estimate(rowCount));
-            }
-
+            double rowCount = Math.max(leftStats.getOutputRowCount(), rightStats.getOutputRowCount()) * JOIN_MATCHING_COEFFICIENT;
```

Comment:
In the commit message title:
Use doubles instead *of* Estimates in PlanNodeStatsEstimate

And at the end of the commit message "connectors" is spelled incorrectly (there's an extra 'e')